examples/avaritia.groovy

```diff
@@ -0,0 +1,15 @@
+
+mods.avaritia.ExtremeCrafting.removeByOutput(item('avaritia:resource', 6))
+mods.avaritia.ExtremeCrafting.shapedBuilder()
```

Comment:
should get examples for a shapeless builder, string array and key replacement, and recipe functions/actions